mofacts/client/views/experiment/card.js

```diff
@@ -1579,7 +1578,7 @@ async function showUserFeedback(isCorrect, feedbackMessage, afterAnswerFeedbackC
         feedbackMessage = "<b style='color:" + uiIncorrectColor + ";'>Incorrect.</b><br>" + feedbackMessage;
```

Comment:
this is not that easy to maintain or understand... also isn't it redundant with the new code? Can we strip it out safely

Reply:
this is 1573 to 1579. I can now set this with displayUserAnswerInFeedback

Reply:
@JRustyHaner @MegaGeese Still wondering about 1573-1579... please explain when they are needed. I don't see why I want to make timeouts different, but only when in the center. Why is this?